settingslib.php

```diff
@@ -0,0 +1,51 @@
+<?php
+// This file is part of Moodle - http://moodle.org/
+//
+// Moodle is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// Moodle is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with Moodle.  If not, see <http://www.gnu.org/licenses/>.
+
+/**
+ * Extend admin_setting_configtext to validate form data in tincanlaunch global settings
+ *
+ * @package    mod_tincanlaunch
+ * @copyright  2013 Andrew Downes
```

Comment:
Should probably update the year, given its a new file. I think the copyright 'Andrew Downes' is right because you are contributing to my repo. Feel free to add an ```@author David Pearce``` row too.